LiteCore/Storage/RecordEnumerator.hh

```diff
@@ -25,7 +25,8 @@ namespace litecore {
 
     enum ContentOptions {
         kDefaultContent = 0,
-        kMetaOnly = 0x01
+        kMetaOnly = 0x01,
+        kHasAttachmentsOnly = 0x02
```

Comment:
ContentOptions specify which content to retrieve from a document, not which documents to return; so this should be a field of RecordEnumerator::Options instead, like `onlyBlobs`.